fast-list.js

```diff
@@ -231,6 +234,7 @@ FastList.prototype = {
 
     function renderItem(i) {
       var item = items[i];
+
```

Comment:
Genius, good catch! Can you think or a more elegant way of replacing items? Are we happy to expect that users might use `.populateItem()` as a hook for switching out nodes?